front/migrations/20240903_backfill_slack_whitelisted_domains_groups.ts

```diff
@@ -0,0 +1,75 @@
+import { EnvironmentConfig } from "@dust-tt/types";
+import { QueryTypes, Sequelize } from "sequelize";
+
+import { Authenticator } from "@app/lib/auth";
+import { Workspace } from "@app/lib/models/workspace";
+import { GroupResource } from "@app/lib/resources/group_resource";
+import { makeSId } from "@app/lib/resources/string_ids";
+import { makeScript } from "@app/scripts/helpers";
+
+makeScript({}, async ({ execute }, logger) => {
+  const connectorDB = new Sequelize(
+    EnvironmentConfig.getEnvVariable("CONNECTORS_DATABASE_URI")
+  );
```

Comment:
You can use `getConnectorReplicaDbConnection`.

Reply:
I can't because I am updated the connectors DB at the end.